test/functional-tests/player.test.js

```diff
@@ -87,6 +87,7 @@ describe('Player:', function () {
   describe('portal player', () => {
     it('plays a video automatically', () => {
       browser.url(`http://localhost:8080/play/${videoId}`)
+      browser.pause(1000)
```

Comment:
Can this be rewritten w/o pause? I don't think I've seen issues with this test failing